src/hooks/useOnboardingFlow.ts

```diff
@@ -41,7 +42,7 @@ function useOnboardingFlowRouter() {
                 return;
             }
 
-            if (NativeModules.HybridAppModule && isLoadingOnyxValue(isSingleNewDotEntryMetadata)) {
+            if (HybridAppModule.isHybridApp() && isLoadingOnyxValue(isSingleNewDotEntryMetadata)) {
```

Comment:
Potentially we can store value of `HybridAppModule.isHybridApp()` to avoid 3 calls to native side here